djangocms_link/forms.py

```diff
@@ -18,13 +26,14 @@ def for_site(self, site):
         # current site
         # this will work for PageSelectFormField
         from cms.models import Page
-        self.fields['internal_link'].queryset = Page.objects.drafts().on_site(site)
+        self.fields['internal_link'].queryset = Page.objects.on_site(site)
```

Comment:
This i where the compatibility code should be.

Reply:
> Hi @crydotsnake, I saw a notification from you today but can't seem to find it.
> 
> Can you please add a new test requirements file for django cms v4 to see what is failing in this project: https://github.com/django-cms/djangocms-link/blob/master/tests/requirements/dj22_cms38.txt
> 
> New file called dj22_cms40.txt, the django-cms version can be controlled by simply adding:
> 
> ```
> -r base.txt
> 
> Django>=2.2,<3.0
> https://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms
> ```
> 
> Then all we need to do is simply add the following entry in the github actions `dj22_cms40.txt`:
> https://github.com/django-cms/djangocms-link/blob/2e934cf4bb3d114ebf66ec70b12502f459e4f328/.github/workflows/test.yml#L14

I only can add the commits too the master branch.. because in the feature branch i dont have the .github folder....